liblangutil/SemVerHandler.cpp

```diff
@@ -116,7 +116,7 @@ bool SemVerMatchExpression::MatchComponent::matches(SemVerVersion const& _versio
 				cmp = static_cast<int>(_version.numbers[i] - version.numbers[i]);
 			}
 
-		if (cmp == 0 && !_version.prerelease.empty() && didCompare)
+		if (cmp == 0 && levelsPresent == 3 && !_version.prerelease.empty() && didCompare)
```

Comment:
Can you add a test case specifically for the `0.8.13-develop...` case you're fixing?